CONTRIBUTING.md

```diff
@@ -127,4 +127,4 @@ All team members are notified when mentioned or assigned to a relevant issue or
 
 We are not overly strict with inviting people to the organization. If you have read the [team permissions](#permissions) and think having additional access would enhance your contributions, please reach out to an [admin](#structure) of the team.
 
-The admins occasionally invite contributors directly if we believe having them on a team will accelerate their work.
\ No newline at end of file
+The admins occasionally invite contributors directly if we believe having them on a team will accelerate their work.
```

Comment:
?

Reply:
File is currently missing a newline at the end of the file, this adds it.

Github is terrible for these kinds of changes.

Reply:
Not sure what editor @qcasey was using for https://github.com/paperless-ngx/paperless-ngx/commit/e1c6042189bd85be7ba4b86b1878d069e8f030ec, but [we are requesting editors to add it](https://github.com/paperless-ngx/paperless-ngx/blob/19ff8339bed01f2030fda73e0021a603ab5b90cb/.editorconfig#L8) (and we should).

Reply:
Ah cool thanks. Ok